pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -50,6 +54,86 @@ def as_matrix(self, time):
     def check_transforms(self):
         self._A2B = check_transform(self._A2B)
         return self
+    
+
+class PandasTimeseriesTransform(TimeVaryingTransform):
```

Comment:
Alternative naming candidates:

- TimeVaryingTransformAsDataFrame
- TimeVaryingTransformPandas
- TransformTimeseriesAsDataFrame

etc.

Reply:
I think the name is good. I cannot think of anything better at the moment.